modules/cid_checker/variables.tf

```diff
@@ -0,0 +1,38 @@
+
+variable "bucket_name" {
+  description = "Name of S3 bucket."
+  type = string
+}
+
+variable "force_destroy" {
+  description = "If `true`, will destroy S3 bucket. Defaults to `false`."
+  type = bool
+  default = false
```

Comment:
As I suggested above, if our default behavior is allowed to destroy, we can probably have the default value for the `force_destroy` as `true`.

Reply:
Removed variable force_destroy.